releng/p2/pom.xml

```diff
@@ -13,7 +13,6 @@
 	</parent>
 
 	<properties>
-		<tycho-version>1.2.0</tycho-version>
```

Comment:
Wird die geerbt?

Reply:
it always was. the p2 generator gradle plugin we used before was just too stupid to know that.